packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -417,6 +419,19 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// {@endtemplate}
   final bool transitionBetweenRoutes;
 
+  /// {@template flutter.cupertino.CupertinoNavigationBar.enableBackgroundFilterBlur}
+  /// Whether to have blur effect with transparent color.
+  ///
+  /// If the [CupertinoNavigationBar] is not a child of a [CupertinoPageScaffold],
+  /// this has no effect.
```

Comment:
I don't believe this is true. I think the blurring happens anytime there is a background color that is not 100% opaque. This can come from either the `CupertinoPageScaffold` background color or the background color of the navbar. Should maybe add an explanation of how this value interacts with `automaticBackgroundVisibility` as well.

Reply:
Yes, I changed the documentation from transparent color to non opaque color. And added documentation for its interaction with `automaticBackgroundVisibility`

Reply:
> If the [CupertinoNavigationBar] is not a child of a [CupertinoPageScaffold], this has no effect.

This logic looks independent of `CupertinoPageScaffold`. The blur will still be added without the scaffold, and with this PR you don't need the scaffold for this property to disable the blur.

Reply:
@MitchellGoodwin removed the line.